app/CreateLocations.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactElement } from "react";
+import { Wrapper } from "./ui/Wrapper";
+import { Button } from "react-bootstrap";
+import { greenActiveButton, greyInactiveButton } from "./ui/colourLibrary";
+
+
+export function CreateLocations(props:{onCreateLocationClick:()=>void, onCancelClick:()=>void}): ReactElement {
+  const {onCreateLocationClick,onCancelClick}=props
+  return <Wrapper>Create Locations
+    <div>
+    <Button
+    onClick={()=>{onCancelClick()}}
+ style={{
+  height: 32,
+  width: 150,
+  margin: 10,
+  borderRadius: 12,
+  fontSize: 12,
+  backgroundColor: greyInactiveButton}}>
+  Cancel
+</Button>
+<Button
+onClick={()=>{onCreateLocationClick()
+    console.log("dummy location creation")}}
+ style={{
+  height: 32,
+  width: 150,
+  margin: 10,
+  borderRadius: 12,
+  fontSize: 12,
+  backgroundColor:greenActiveButton}}>
+  Create
+</Button>
+</div>
+  </Wrapper>
+}
```

Comment:
we really need to set up `format on save` on your desktop.

Why not just give your large button extra configuration like the ability to specify background colour?

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/14/commits/52bee702893f4be9acf0179814fe759697d6d755